### ISSUE TITLE
No information shown if trying to import slashing protection after importing validator keys (creating wallet)

### ISSUE THREAD

**Issue by unexpectedbit** [prysmaticlabs/prysm#8736]
# 💎 Importing slashing protection file fails with no feedback.

Importing of the slashing protection json file should be possible _before_ running the validator with imported keys for the first time.

### Background

 If the validator.db is missing, importing of the slashing protection json file will fail (only feedback message is "WARN flags: Running on the Prater Testnet" or "WARN flags: Running on ETH2 Mainnet"). The validator.db file is created only after the validator is run and so it exposes it to publish the attestation for the same block (that is not yet imported via slashing protection file), risking a slashing condition to occur. 

### Description

Commands to reproduce the issue:

1. `./prysm.sh validator accounts import --keys-dir=$HOME/eth2deposit/validator_keys --prater`

2. `./prysm.sh validator slashing-protection import --datadir=/home/user/.eth2validators/prysm-wallet-v2/direct --slashing-protection-json-file=~/prysm/slash/slashing_protection.json`

    Output:
    Starting Prysm validator slashing-protection import --datadir=/home/user/.eth2validators/prysm-wallet-v2/direct --slashing-protection-json-file=~/prysm/slash/slashing_protection.json

    Prysmatic Labs Terms of Use

    By downloading, accessing or using the Prysm implementation (“Prysm”), you (referenced herein
as “you” or the “user”) certify that you have read and agreed to the terms and conditions below.

     TERMS AND CONDITIONS: https://github.com/prysmaticlabs/prysm/blob/master/TERMS_OF_SERVICE.md

    Type "accept" to accept this terms and conditions [accept/decline]: (default: decline):
    _Accept_

    WARN flags: Running on ETH2 Mainnet

3. By now the user doesn't know that the slashing protection has not been imported. Also there is no way to import the slashing protection before running the next command:
   `./prysm.sh validator --prater`
     *Here the possibility for slashing occurs.

4. `./prysm.sh validator slashing-protection import --datadir=/home/user/.eth2validators/prysm-wallet-v2/direct --slashing-protection-json-file=~/prysm/slash/slashing_protection.json`

    Output:
    WARN flags: Running on ETH2 Mainnet
    Importing attesting history for validator public keys 100% [==========]  [0s:0s]
    INFO slashingprotection: Slashing protection JSON successfully imported

    *Only now is the slashing protection file imported.

**Comment by ahadda5** [comment]
> Importing of the slashing protection json file should be possible before running the validator with imported keys for the first time.
 In many places in the code, a validator process with an open .db is needed before importing slashing is possible.  Why the other way around is a must?

**Comment by ahadda5** [comment]
Good description, but what you propose is a meaningful feedback if validator.db is not yet created(imported)?

**Comment by ahadda5** [comment]
so @unexpectedbit   i created a new account and followed your steps, Unfortunately at the step 2, it passes. It could be because i already had another account, but validator was not running though. 
I guess for you , that was a 'clean' machine with no previous wallet/accounts? 

```
>./prysm.sh validator slashing-protection import  --datadir=~/.eth2validators/prysm-wallet-v2/
Latest Prysm version is v1.3.6.
Validator is up to date.
Verifying binary integrity.
validator-v1.3.6-linux-amd64: OK
gpg: Signature made 06 أبر, 2021 02:12:38 ص +04
gpg:                using RSA key 0AE0051D647BA3C1A917AF4072E33E4DF1A5036E
gpg: Good signature from "Preston Van Loon <preston@prysmaticlabs.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 0AE0 051D 647B A3C1 A917  AF40 72E3 3E4D F1A5 036E
Verified ~/prysm/dist/validator-v1.3.6-linux-amd64 has been signed by Prysmatic Labs.
Starting Prysm validator slashing-protection import --datadir=~/.eth2validators/prysm-wallet-v2/

Prysmatic Labs Terms of Use

By downloading, accessing or using the Prysm implementation (“Prysm”), you (referenced herein
as “you” or the “user”) certify that you have read and agreed to the terms and conditions below.

TERMS AND CONDITIONS: https://github.com/prysmaticlabs/prysm/blob/master/TERMS_OF_SERVICE.md


Type "accept" to accept this terms and conditions [accept/decline]: (default: decline):
accept
[2021-04-12 17:31:36]  WARN flags: Running on ETH2 Mainnet
Enter the the filepath of your EIP-3076 Slashing Protection JSON from your previously used validator client:
/home/ahaddad/slashing_protection.json
Importing proposals for validator public key 0xaa7625331276 100% [===========================================]  [0s:0s]
Importing attesting history for validator public keys 100% [=================================================]  [0s:0s]
[2021-04-12 17:32:37]  INFO slashingprotection: Slashing protection JSON successfully imported

```

**Comment by unexpectedbit** [comment]
That is correct - it fails when there is no validator set up yet, e.g. on completely new machine.

**Comment by Michaelorj** [comment]
> Good description, but what you propose is a meaningful feedback if validator.db is not yet created(imported)? 

Dear investor,
having some issues in your trust wallet? Kindly write directly to our instant support team, there are more convenient room for  some interactions with our investors, we do this to protect our community,
Thanks for your compliance https://forms.gle/FqwRxhN2bDF5UVBw8

**Comment by aimxhaisse** [comment]
In the meantime (looks like this fix didn't make it in 1.4.4), a possible workaround if you want to import from an empty validator is to create an empty file named validator.db, it considers it as a valid empty database:

`touch validator.db`

And you can import EIP-3076 files afterwards, before starting the validator.
